content/release-engineering/cicd-process-steps/examples/deploy-with-codefresh.yaml

```diff
@@ -0,0 +1,31 @@
+version: '1.0'
+steps:
+  deploy_helmfile:
+    title: Deploy app with helmfile
+    image: cloudposse/build-harness:${{BUILD_HARNESS_VERSION}}
+    environment:
+    - AWS_REGION=us-west-2
+    - AWS_ACCESS_KEY_ID=${{AWS_ACCESS_KEY_ID}}          # for chamber
+    - AWS_SECRET_ACCESS_KEY=${{AWS_SECRET_ACCESS_KEY}}  # for chamber
+    - CHART_NAME=app
+    - CHART_REPO_URL=${{STAGING_REPO_ENDPOINT}}
+    - CHART_VERSION=${{SEMVERSION_COMMIT_SHORT}}
+    - EXTERNAL_DNS_ALPHA_KUBERNETES_IO_TARGET=ingress.${{BASE_HOST}}
```

Comment:
Where is this used? Not here: https://github.com/cloudposse/geodesic/blob/master/rootfs/conf/kops/helmfile.yaml